### DISCUSSION TITLE
[QUESTION] Roadmap for m88 and build stability

### DISCUSSION THREAD

**Discussion by Danielku15** [mono/SkiaSharp#1644] [Q&A]
Hi there. 

Sorry to bother you with this topic again directly after you released the Skia m80 based SkiaSharp. The m80 update solved [long runner issue](https://github.com/mono/SkiaSharp/issues/1253) for me. I highly appreciate that 🎉 👍 . 

Unfortunately m80 suffers from some "edgy" rendering of quadratic curves which causes some issues on my side with my visual regression test suite. Here the concrete example: The m80 curve seems to be rendered quite edgy instead of a smooth curve. This mainly happens on small resolutions:
![image](https://user-images.githubusercontent.com/674916/109397223-fffe1000-7935-11eb-9841-4a30828ca937.png)

Chrome m80 suffered from the same issue so it is just a matter of having the latest Skia integrated. I saw that you [just lately](https://github.com/mono/SkiaSharp/commit/c311edea87e21c97a3fd4d9cc96cdf8497258d4f) updated on the `develop` branch to m88 and I was wondering if you already have a first guess on when you will release a potential 2.88? There is not yet any milestone defined but the development seems to be quite active.

Also I wanted to contribute some changes related to #1253 where the text align is not respected when using Harfbuzz. Unfortunately it seems that the current `develop` state is not testable. All PRs are currently red and locally fetching the pre-built native libs cause runtime issues. Do you have a potential ETA when `develop` will be stable enough again for other devs to contribute. I followed the "Managed-Only Building" guide from the Wiki but the latest native DLLs are still on m80. 

Keep up the good work. 

Greetings Daniel

**Comment by mattleibow** [comment]
Hey there. Sorry about the long delay of nothing happening on this repo. I was just doing crazy things with all the new bits in MAUI for preview 2 of .NET 6. Luckily, with the previews of .NET 6, we now support the M1 chip... Which means I am "forced" to get back onto SkiaSharp to release m88 which has M1 support 😉 

I just need to review what I had left and then maybe get a preview out.
